syntax = "proto2";

message ReclaimLSK {
    required uint64 amount = 1;
    }

